code/obj/machinery/clonepod.dm

```diff
@@ -281,12 +281,15 @@ TYPEINFO(/obj/machinery/clonepod)
 			 */
 			var/generation = src.occupant.bioHolder.clone_generation
 			for (var/i in 1 to rand(round(generation / 2)  * (src.emagged ? 2 : 1), (generation * (src.emagged ? 2 : 1))))
-				if (generation)
+				if (generation || traits.hasTrait("defect_prone"))
```

Comment:
```suggestion
				if ((generation > 1) || traits.hasTrait("defect_prone"))
```
Per [a discussion on Discord](https://discord.com/channels/182249960895545344/890226559691157524/1399827450728550461), we've discovered that `generation` here is always 1 or greater, so `if (generation)` will always run.